### ISSUE TITLE
Is bidder really blocking my ad server?

### ISSUE THREAD

**Issue by polapi** [prebid/Prebid.js#1364]
## Type of issue
Question

## Description
I've been looking into optimizing my prebid.js setup using chrome extension. Usually the tool shows problem described in [scenario 3](http://prebid.org/overview/optimal-header-bidding-setup.html#scenario-3) of the guide. That case:
<img width="510" alt="adserver-too-late" src="https://user-images.githubusercontent.com/15109076/28065534-4024e8b6-6639-11e7-9fba-27c288d93779.png">

The help hint for such case says:
_Set a shorter timeout for the page and remove any bidder that is blocking the ad server from loading._ 

I assume the problem is caused by bidder taking to long to process the response. Is it correct? I don't think that processing those responses should be taking long. I am using bidders which got code reviewed and are included in prebid.js repository. Is really such problem caused by bidders blocking ad server? How can I check which one?

**Comment by mkendall07** [comment]
It doesn't look like any bidders are blocking DFP. They are all returning quickly based on the graph above. The more likely reason is other content on the page is blocking DFP from loading. 

Have you tried loading DFP and Prebid.js concurrently? This is the example page: https://github.com/prebid/Prebid.js/blob/master/integrationExamples/gpt/load_pbjs_dfp_concurrently.html

**Comment by polapi** [comment]
Umm, not sure why exactly, the problem got itself solved. I did do some minor refactoring and upgraded Prebid.js after that it seems to work better. But maybe it could've been something on DSP end. 

Anyway, thanks for the tip with loading concurrently. I also gave it a try but it doesn't look that's going to give significant improvement. Prebid.js is being browser cached, so making it load slightly faster for the first page view seems to be a little bit over optimization, I think.
